packages/vue/src/components/range/RangeSlider.jsx

```diff
@@ -129,14 +130,22 @@ const RangeSlider = {
     this.setQueryListener(this.$props.componentId, onQueryChange, null);
   },
   beforeMount(){
+
     this.addComponent(this.$props.componentId);
+    this.addComponent(this.internalComponent);
     this.setReact(this.$props);
-    if (this.selectedValue) {
-      this.handleChange(this.selectedValue);
-    } else if (this.$props.defaultSelected) {
-      this.handleChange(this.$props.defaultSelected);
+
+    const { selectedValue, defaultSelected } = this.$props;
```

Comment:
How you're accessing this `selectedValue` from `this.$props`, afaik it's coming from redux.

Reply:
@bietkul yes it is coming as a prop from here - 
```
  selectedValue: state.selectedValues[props.componentId]
    ? state.selectedValues[props.componentId].value
    : null,
```

Reply:
Updated